.github/workflows/tests.yml

```diff
@@ -49,7 +49,12 @@ jobs:
         run: ./bin/run_clang_format.sh
       - name: "Check C/C++ formatting changes"
         run: git diff --exit-code
-      # --- Compilation ---
+      # --- Code build ---
+      - name: "Rerun cmake"
+        run: inv -r faasmcli/faasmcli dev.cmake
+      - name: "Build dev tools"
+        run: inv -r faasmcli/faasmcli dev.tools
```

Comment:
Better to use the same commands in the CLI and tests to build the code (it'll catch it if they break)